### PR TITLE
Removed unused variables

### PR DESCRIPTION
**Pull request by webimpress**
These variables are not used, can be safely removed.

### CODE REVIEW

**Comment by webimpress** [comment]
Rebased, conflicts resolved.

File: src/Standards/PEAR/Sniffs/Commenting/ClassCommentSniff.php [hunk 1]
```diff
@@ -45,9 +45,7 @@ public function process(File $phpcsFile, $stackPtr)
     {
         $this->currentFile = $phpcsFile;
 
-        $tokens    = $phpcsFile->getTokens();
-        $type      = strtolower($tokens[$stackPtr]['content']);
```
Comment (jrfnl) [inline src/Standards/PEAR/Sniffs/Commenting/ClassCommentSniff.php]: :question:  Looks like `$type` and `$errorData` should be used (in the error message), but the usage got accidentally removed when the new comment tokenizer got implemented. Relevant commits: 371ec577d810899b6e1122f7e8118405b6a7de53, 5292c84edf3d1c00610df34362397825a159ed3a and 22060b65a3cb54a49011d87fd90cb3dac5388746

File: src/Ruleset.php [hunk 2]
```diff
@@ -946,8 +946,6 @@ private function processRule($rule, $newSniffs, $depth=0)
                         $value  = (string) $prop['value'];
                         $values = array();
                         foreach (explode(',', $value) as $val) {
-                            $v = '';
```
[the remark this answers is not in the document]
Comment (webimpress) [reply src/Ruleset.php]: `$v` is defined in the next line (949) with function `list`

File: src/Standards/Generic/Sniffs/Debug/ClosureLinterSniff.php [hunk 3]
```diff
@@ -71,9 +71,8 @@ public function process(File $phpcsFile, $stackPtr)
 
         $fileName = $phpcsFile->getFilename();
 
-        $lintPath = escapeshellcmd($lintPath);
-        $cmd      = '$lintPath --nosummary --notime --unix_mode '.escapeshellarg($fileName);
-        $msg      = exec($cmd, $output, $retval);
+        $cmd = '$lintPath --nosummary --notime --unix_mode '.escapeshellarg($fileName);
```
[the remark this answers is not in the document]
Comment (webimpress) [reply src/Standards/Generic/Sniffs/Debug/ClosureLinterSniff.php]: Oh... yeah, you have right. I didn't notice that. I think there should be double quotes instead then, or concatenation... Changed in 0522799ffe872b4524aab5b301b8ea89f6fb5db0

**Comment by gsherwood** [comment]
I finally got around to reviewing this. Since you submitted, I have fixed a few of these just in the course of other work, and some code has moved around, but I found a lot of unused vars that could be removed.

This also picked up that the gjslint sniff wasn't working (I committed the fix for that separately) and that the class doc comment sniff wasn't using the object name (it was hard-coding class) so I fixed that as well.

Thanks for this PR, and sorry I didn't get around to it sooner and was able to merge it in.

**Comment by webimpress** [comment]
@gsherwood it's ok, fine we have it fixed now! Thanks :+1:

**Pull request closed without merging**